tests/src/ut_test_control_plane.c

```diff
@@ -321,6 +328,18 @@ void test_ut_control_performExit( void )
     UT_ControlPlane_Exit(gInstance);
 }
 
+void test_ut_control_get_map_value( void )
+{
+    UT_LOG("test_ut_control_get_map_value()\n");
+    UT_ASSERT(UT_Control_GetMapValue(numericMaptable, "three", 0) == 3);
+}
+
+void test_ut_control_get_map_string( void )
+{
+    UT_LOG("test_ut_control_get_map_string()\n");
+    UT_ASSERT_EQUAL(UT_Control_GetMapString(numericMaptable, 1), "one");
```

Comment:
`UT_ASSERT_EQUAL_STRING()` am surprised this worked, I don't think it should.

Reply:
are you not getting compiler warnings? Is all the warnings enabled in the makefile?

Reply:
done